### PR TITLE
about: Add introduction and history

### PR DESCRIPTION
**Pull request by melvo**

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -2,6 +2,38 @@
...
+All properties of the included providers have references in order to verify and maintain them.
+Therefore, it is important that providers add corresponding statements to their websites.
+
+This website uses XMPP Providers to generate its provider listing, show the detail pages for individual providers and explain how a provider could improve its category.
+Help us to make _free communication_ possible and [contribute](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/CONTRIBUTING.md) to this project!
+
+## History
+
+XMPP Providers arose from [Kaidan](https://www.kaidan.im)'s onboarding introduced in [Kaidan 0.5](https://www.kaidan.im/2020/04/06/kaidan-0.5.0/).
+The goal was to speed up the onboarding process by making registrations as simple as possible.
```
Comment (Echolon) [inline content/about.md:25]: I think here its okay to say something alike "From here the authors, names.., developed and reviewed meadures to bring the project to the next level."

Comment (Echolon) [reply content/about.md:25]: I think we should also say that automated onboarding requires qualified server to ensure a good user experience. Furthermore, it provides more transparency and review of quality.

File: content/about.md [hunk 2]
```diff
@@ -2,6 +2,38 @@
...
+XMPP (E**x**tensible **M**essaging and **P**resence **P**rotocol) is like a common language your and your chat partner's app speak to each other.
+Similar to email, there is a service provider managing your account and messages.
+That provider speaks XMPP too.
+
+But some XMPP providers support functionalities that other providers do not.
+Some support registrations via your app and some only via their websites.
+Some provide more space for sharing media than others or store them longer than others.
+The **XMPP Providers** makes it easy to find a suitable provider.
+
+All properties of the included providers have references in order to verify and maintain them.
```
Comment (wurstsalat3000) [inline content/about.md:16]: this is vague

File: content/about.md [hunk 3]
```diff
@@ -2,6 +2,38 @@
...
+Therefore, it is important that providers add corresponding statements to their websites.
+
+This website uses XMPP Providers to generate its provider listing, show the detail pages for individual providers and explain how a provider could improve its category.
```
Comment (wurstsalat3000) [inline content/about.md:19]: This website uses **the** XMPP Providers **project** ? (+ link) to generate its provider listing, ~~show~~ the detail pages for each provider **,** and **to** explain how ...

File: content/about.md [hunk 4]
```diff
@@ -2,6 +2,38 @@
...
+
+Until then, a usual onboarding raised the following questions:
+1. Which *username* should I take?
+1. Which *password* should I take?
+1. Which *provider* should I take?
+
+We wanted to tackle the third question by creating a curated list of providers.
+There were several lists on the web but none was really curated.
+Furthermore, there were no criteria for recommended providers.
+Thus, we created a list containing all properties of providers relevant to the users during the onboarding.
```
Comment (wurstsalat3000) [inline content/about.md:35]: ..onboarding **process**.

**Review by Echolon — APPROVED** [review]

**Pull request merged**